core/advancedpolls.php

```diff
@@ -346,6 +352,23 @@ public function do_poll_modification($topic_data)
 	}
 
 	/**
+	 * Add the possible options to the template
+	 *
+	 * @param array	$post_data		The array of post data
+	 * @return void
+	 */
```

Comment:
Ok, I will.  Take a look at the other docblocks in the file (yours): one is
misplaced (duplicate of another), and another is a copy&paste of another
ext.

2015-04-07 3:46 GMT+02:00 Wolfsblvt <notifications@github.com>:

> In core/advancedpolls.php
> <https://github.com/Wolfsblvt/advancedpolls/pull/17#discussion_r27846471>:
>
> > @@ -346,6 +352,23 @@ public function do_poll_modification($topic_data)
> >  	}
> >
> >  	/**
> > +	 * Add the possible options to the template
> > +	 *
> > +	 * @param array	$post_data		The array of post data
> > +	 * @return void
> > +	 */
>
> Please update this docblock with a short explaining sentence what this
> function does/what it is for and the correct parameter name, data type and
> description.
>
> That helps me to understand the code months and years later :P And my IDE
> also shows the text on IntelliSense.
>
> Ah, and while you are at it. Even an internal function like the
> order_by_votes() below should have a docblock. parameters of course, and
> maybe a sentence like "Function used for sorting blah bblahb blah"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wolfsblvt/advancedpolls/pull/17/files#r27846471>.
>